src/diffusers/__init__.py

```diff
@@ -30,12 +30,15 @@
     )
     from .pipeline_utils import DiffusionPipeline
     from .pipelines import (
+        AudioDiffusionPipeline,
         DanceDiffusionPipeline,
         DDIMPipeline,
         DDPMPipeline,
         KarrasVePipeline,
+        LatentAudioDiffusionPipeline,
         LDMPipeline,
         LDMSuperResolutionPipeline,
+        Mel,
```

Comment:
Could we remove this from the general `__init__.py` function? -> I don't think one would use `"Mel"` witouth the pipelines no? :-)

Reply:
A mel instance is a parameter to the pipeline and it is useful for creating the dataset and training the model. I agree that it will probably only be used in conjunction with the pipelines, so as long as you can import it from diffusers.pipelines that should be Ok. Is that what you mean? Thanks!